### PR TITLE
PAE-628-3 - Fix MIT Horizon TypeError

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
Fix a TypeError

### CODE REVIEW

File: openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def _get_bearer_token(self):
                 return 'missing token'
 
             return response.json()['access_token']
+        return ''
```
Comment (ivanvgh) [inline openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py:73]: Is it right to return an empty string? @luismorenolopera

Comment (luismorenolopera) [reply openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py:73]: Yes, that's fine. Later it detects that it is not a valid token. But I think it is better to add a more descriptive message to this string @ivanvgh

**Review by ivanvgh — APPROVED** [review]

File: openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def _get_bearer_token(self):
             )
         else:
             if not response.ok:
-                log_details['response'] = response.text()
+                log_details['response'] = response.text
```
Comment (diegomillan) [inline openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py:64]: Is there any reason why we are not using response.json() @luismorenolopera ?

**Review by diegomillan — CHANGES_REQUESTED** [review]

File: openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py (hunk 2 above)
Comment (luismorenolopera) [reply openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py:64]: Yes.  the MIT Horizon API when it fails does not return a JSON, it returns an HTML file

**Review by diegomillan — APPROVED** [review]

**Pull request merged**